Dockerfile

```diff
@@ -31,7 +31,7 @@
 #   2. ❯ docker run --rm -p 3000:3000 -t qiskit-docs-preview
 #   3. Open up http://localhost:3000
 
-FROM icr.io/quantum-computing/iqp-channel-docs-dev:next13.4
+FROM icr.io/quantum-computing/iqp-channel-docs-dev:hostname-experiment
```

Comment:
Is this a permanent name, or just while we test it out?

Reply:
I was using it to experiment, then was too lazy to change it. I can change it if we want to `latest`?